catalog/opensearch/catalog-opensearch-source/src/test/java/org/codice/ddf/opensearch/source/OpenSearchParserImplTest.java

```diff
@@ -56,12 +58,17 @@
 
   private static final GeometryFactory GEOMETRY_FACTORY = new GeometryFactory();
 
+  private static final WKTReader WKT_READER = new WKTReader();
```

Comment:
❓ Since it looks like `WKTReader` uses mutable instance variables in its source code I'm not sure if its threadsafe. Is it okay to have a single instance of this (and a static instance at that) ?